### ISSUE TITLE
error: unrecognized arguments

### ISSUE THREAD

**Issue by LBlend** [domeneshop/certbot-dns-domeneshop#12]
Trying to use your plugin with the example listed in the README yields the following error

```
certbot: error: unrecognized arguments: --dns-domeneshop-credentials /path/to/domeneshop.ini --dns-domeneshop-propagation-seconds 120
```

running `certbot plugins` shows that I do ineed have the plugin installed

```
$ certbot plugins

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
* dns-domeneshop
Description: Obtain certificates using a DNS TXT record (if you are using
Domeneshop for DNS).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='dns-domeneshop',
value='certbot_dns_domeneshop.dns_domeneshop:Authenticator',
group='certbot.plugins')
```

I am running pip package version is `certbot-dns-domeneshop 0.2.9` and I am running certbot version `2.10.0`

**Comment by eirikrye** [comment]
I am unable to reproduce this with `certbot==2.10.0`, unfortunately.

```
(certbot) % pip3 freeze | grep certbot
certbot==2.10.0
certbot-dns-domeneshop==0.2.9
```

Are you certain first error was produced from the same environment as your `certbot plugins` output? The error you're seeing is exactly the same error that is produced when the plugin is *not* available:

```
(certbot) % pip3 uninstall certbot-dns-domeneshop
Found existing installation: certbot-dns-domeneshop 0.2.9
[...]
Proceed (Y/n)? y
  Successfully uninstalled certbot-dns-domeneshop-0.2.9
(certbot) % certbot certonly --authenticator dns-domeneshop --dns-domeneshop-credentials /path/to/domeneshop.ini --dns-domeneshop-propagation-seconds 120 -d example.com
[...]
certbot: error: unrecognized arguments: --dns-domeneshop-credentials /path/to/domeneshop.ini --dns-domeneshop-propagation-seconds 120
```

**Comment by LBlend** [comment]
This seems like a plausible explanation as I'm using pyenv. I tried fiddling around with it to get it working but wasn't able to, which is entirely a fault on my part. Not that I put much effort into trying before giving up though....

I ended up trying out docker and spending my time debugging why the docker solution wasn't working. Turns out snap packages suck. Who would've thoguht!

**Anyway** enough with the rant on my part. I managed to run it on docker instead and your explanation seems to be the root cause of the problem,

Thank you for taking your time to help!
